README.md

```diff
@@ -27,10 +27,26 @@ rails db:migrate
 ### Usage
 Start server with:
 
-```rails server```
+```
+rails server
+```
 
 Open http://localhost:3000/ in your browser.
 
+### Run tests
+Install rspec with:
+```
+bundle install
+```
+and
+```
+rails generate rspec:install
+```
+run the test with:
+```
+rspec spec
+```
```

Comment:
- [ ] Please mention the steps to install the `npm` packages so that if anyone can test it will pass. [`npm i`]

Reply:
Hi @DeliceLydia, Thanks for reviewing my pull request. Ok, I will mention it.